src/server.js

```diff
@@ -25,4 +25,4 @@ server.register([Inert], (err) => { // We register extra modules like inert here
   server.route(routes); // Plug the routes we imported earlier into the server
 });
 
-module.exports = server; 
+module.exports = server;
```

Comment:
This looks like whitespace that has been caught by an automatic linter. Ideally you wouldn't have this showing up as a change to your code in this pull request (since there's no actual change to this file).

Can I suggest that either both of you turn the automtic linting on, or both of you turn this off? Otherwise one person's automatic linting will catch things that the other person has missed :+1: